script/create_contributor_list.py

```diff
@@ -6,7 +6,7 @@
 
 from contributors_txt import create_contributors_txt
 
-CWD = Path(".").absolute()
+CWD = Path().absolute()
```

Comment:
I actually prefer the old version but that might be personal preference.

Reply:
Yeah, you have to know the default value so implicit vs explicit. The worst check is the only one autofixxed.